packages/plugins/workspace-search/example/web-imports/package.json

```diff
@@ -1,19 +1,18 @@
 {
-  "name": "blockly-ws-search-web-imports",
+  "name": "@blockly/plugin-workspace-search-example-web-imports",
   "version": "1.0.0",
   "description": "Example of workspace search with web imports.",
   "private": true,
   "scripts": {
-    "preinstall": "cd ../.. && npm install",
-    "prepare": "webpack --display-error-details",
-    "clean": "rm $npm_package_directories_dist/main.js",
+    "build": "webpack --display-error-details",
+    "clean": "rm -rf $npm_package_directories_build",
     "start": "webpack-dev-server"
   },
   "directories": {
-    "dist": "dist"
+    "build": "build"
```

Comment:
We should probably create a follow up and update this example to generate files to dist directory instead of build.

Reply:
Build is being used for development (eg: test), and dist for prod. I think it makes sense to leave this one under build since its just a dev package.